tms_objects2.erb

```diff
@@ -536,7 +536,13 @@ xsi:schemaLocation="http://www.lido-schema.org http://www.lido-schema.org/schema
         <% end %>
               <lido:actorInRole>
                 <lido:actor lido:type="<%= Owners[0]["ConstituentType"] %>">
-                  <lido:actorID lido:source="YCBA" lido:type="local">ycba_actor_<%= Owners[0]["ConstituentID"] %></lido:actorID>   
+                  <lido:actorID lido:source="YCBA" lido:type="local">ycba_actor_<%= Owners[0]["ConstituentID"] %></lido:actorID>  
+                  <lido:nameActorSet>
+        <% OwnerAltName.each do |oa| %>
+                    <lido:appellationValue lido:pref="<%= oa["NameType"] %>"><%= oa["DisplayName"] %></lido:appellationValue>
```

Comment:
I don't see the appellationValues coming through like in:

https://harvester-bl.britishart.yale.edu/oaicatmuseum/OAIHandler?verb=GetRecord&identifier=oai:tms.ycba.yale.edu:9&metadataPrefix=lido